mappings/net/minecraft/block/Block.mapping

```diff
@@ -171,6 +178,7 @@ CLASS net/minecraft/class_39 net/minecraft/block/Block
 		ARG 3 y
 		ARG 4 z
 		ARG 5 face
+	METHOD method_121 returnZero ()I
```

Comment:
Is this just a method with a body of `return 0;`? If so, maybe see where it is used. If it's not used anywhere, this name could be acceptable, but otherwise I think it should be changed.

Reply:
I made this change when I it was early in the day. In hindsight, it wasn't a good decision. I'll probably make changes to most of these tomorrow.

Reply:
I'm going to remove the mapping until I figure out what it actually does.